docs/tilesource_options.rst

```diff
@@ -59,15 +59,15 @@ A band definition is an object which can contain the following keys:
 
 - ``nodata``: the value to use for missing data.  null or unset to not use a nodata value.
 
-- ``composite``: either 'lighten' or 'multiply'.  Defaults to 'lighten' for all except the alpha band.
+- ``composite``: either 'lighten' or 'multiply'.  Defaults to 'lighten' for all except the alpha band. Read more about blend modes and see examples `here <blend_modes_>`_.
```

Comment:
We don't use the python library you've linked to.  Maybe the css definitions are better: `https://developer.mozilla.org/en-US/docs/Web/CSS/blend-mode`.